apis/v1beta1/microk8sconfig_types.go

```diff
@@ -69,6 +69,10 @@ type InitConfiguration struct {
 	// The optional IPinIP configuration
 	// +optional
 	IPinIP bool `json:"IPinIP,omitempty"`
+
+	// The confinement (strict or classic) configuration
+	// +optional
```

Comment:
Let's add a kubebuilder rule to only allow `strict` and `classic` as values?

Reply:
Great idea. Thanks. Added.